css/index.css

```diff
@@ -0,0 +1,57 @@
+body {
+    background:rgb(255,160,122);
+}
+
+.grid-container {
```

Comment:
It looks like there's quite a lot of repetition in the CSS files. Try to extract as many of the rules into the general style file, and then only put the differences into per-page style files. Then on the various pages, simply link both the general and the per-page CSS file, with the per-page file coming second. It will override the general style rules, cutting down significantly on the duplication between files.